apps/thunderhub/docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  thub:
+    image: apotdevin/thunderhub:v0.12.0
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 5m
+    ports:
+      - 3000:3000
+    volumes:
+      - ${LND_DATA_DIR}:/lnd:ro
+    environment:
+      NO_VERSION_CHECK: "true"
+      NO_CLIENT_ACCOUNTS: "true"
+      LOG_LEVEL: "debug"
+      SSO_SERVER_URL: "https://${LND_IP}:${LND_GRPC_PORT}"
+      SSO_MACAROON_PATH: "${LND_DATA_DIR}/data/chain/bitcoin/${BITCOIN_NETWORK}"
+      SSO_CERT_PATH: "${LND_DATA_DIR}/tls.cert"
```

It accesses the mount point inside the container, so you can hardcode these paths.
```suggestion
      SSO_MACAROON_PATH: "/lnd/data/chain/bitcoin/${BITCOIN_NETWORK}"
      SSO_CERT_PATH: "/lnd/tls.cert"
```